examples/llava/mtmd-cli.cpp

```diff
@@ -50,8 +52,10 @@ static void sigint_handler(int signo) {
             g_is_generating = false;
         } else {
             console::cleanup();
-            LOG("\nInterrupted by user\n");
-            _exit(130);
+            if(!is_app_running) {
```

Comment:
```suggestion
            if (!is_app_running) {
```

Our code style is always `if` followed by a space, please update in other places in your PR